### ISSUE TITLE
Passing nil closure to ACPTarget.prefetchContent results in crash

### ISSUE THREAD

**Issue by DennisWolfeDSW** [Adobe-Marketing-Cloud/acp-sdks#185]
### Problem

We're seeing a crash in the iOS SDK that seems like it could be related to accessing a null pointer in the C++ portion of the SDK.

### Environment

Xcode Version: 10.2.1 (10E1001)
Adobe SDK Version: 
ACPCore - 2.3.2
ACPTarget - 2.1.1
ACPMobileServices - 1.0.0
Steps to reproduce: Pass nil as an argument for the callback parameter for ACPTarget.prefetchContent.

```
ACPTarget.prefetchContent(..., with: nil, callback: nil)
```

### SDK Integration
We've integrated with the SDK manually instead of using cocoapods. Here is a rough code sample of how we initialize and use the SDK.

```
APCore.setLogLevel(.error)
ACPCore.configure(withAppId: ...)
ACPIdentity.registerExtension()
ACPTarget.registerExtension()
ACPMobileServices.registerExtension()
ACPCore.start()
ACPTarget.prefetchContent(..., with: nil) { _ in }
```

**Comment by rymorale** [comment]
Thanks for the information @DennisWolfeDSW. We will investigate the issue further and will update this ticket if we have ant new information.

**Comment by rymorale** [comment]
Hi @DennisWolfeDSW, we are able to reproduce the crash and are now working on a fix.

Thanks,
Ryan Morales
Mobile Engineer
[Adobe Experience Cloud](https://www.adobe.com/experience-cloud.html)

**Comment by DennisWolfeDSW** [comment]
@rymorale Thanks!

**Comment by rymorale** [comment]
Hi @DennisWolfeDSW,

We released ACPTarget v2.1.3 which should resolve the ACPTarget.prefetchContent crash that you are seeing. Could you give the build a try?

Thanks,
Ryan Morales
Mobile Engineer
[Adobe Experience Cloud](https://www.adobe.com/experience-cloud.html)

**Comment by rymorale** [comment]
@DennisWolfeDSW, I am closing this issue as the fix has been released. Please give the latest 2.1.3 Target release a try and feel free to re-open this issue if the issue still persists.

Thanks,
Ryan Morales
Mobile Engineer
[Adobe Experience Cloud](https://www.adobe.com/experience-cloud.html)
